q2_types/feature_data/tests/test_format.py

```diff
@@ -901,5 +903,37 @@ def test_blast6_format_invalid(self):
             BLAST6DirectoryFormat(temp_dir, mode='r').validate()
 
 
+class TestSequenceCharacteristicsFormat(TestPluginBase):
+    package = 'q2_types.feature_data.tests'
+
+    def test_sequence_characteristics_directory_format(self):
+        filepath = self.get_data_path('sequence_characteristics_length.txt')
+        temp_dir = self.temp_dir.name
+        shutil.copy(filepath, os.path.join(temp_dir,
+                                           'sequence_characteristics.txt'))
```

Comment:
```suggestion
                                           'sequence_characteristics.tsv'))
```

Reply:
Ah, it also looks like there's a file name mismatch here - there isn't a `sequence_characteristics.tsv` file that's been added. Maybe this was supposed to be `sequence_characteristics_length.tsv`?